.github/ISSUE_TEMPLATE/BUG.yml

```diff
@@ -0,0 +1,89 @@
+name: Bug Report
+description: File a bug report
+title: "[Bug]: "
+labels: ["Type:Bug", "Triage:Untriaged"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this bug report!
+
+        If you're having trouble with the NuGet client tools (the Visual Studio extension, NuGet.exe command line tool, etc.), you are in the right place.
+
+        If you're having trouble with the NuGet.org Website, please post in [NuGetGallery issues](http://github.com/nuget/nugetgallery/issues)
+  - type: markdown
+    attributes:
+      value: "# Details About the Problem"
+  - type: dropdown
+    id: product
+    attributes:
+      label: NuGet Product Used
+      multiple: true
+      description: Which of our NuGet clients does this apply to?
+      options:
+        - NuGet.exe
+        - Visual Studio Package Management UI
+        - MSBuild.exe
+        - dotnet.exe
+    validations:
+      required: true
+  - type: input
+    id: version
+    attributes:
+      label: Product Version
+      description: Which software version does this apply to?
+      placeholder: dotnet --info, msbuild -version, nuget.exe ?, etc
+    validations:
+      required: true
+  - type: checkboxes
+    id: regression
+    attributes:
+      label: Worked Before?
+      description: Is this a regression? Did it work in a previous version?
+      options:
+        - label: This used to work.
+    validations:
+      required: false
+  - type: input
+    id: lastworking
+    attributes:
+      label: Which Version?
```

Comment:
Consider just having this in the "Worked Before?" section to reduce headings/bulk.

Reply:
You can't omit labels, unfortunately. I tried to do this because yeah, that's a lot of headings :(